packages/plugins/registry/src/lib/infrastructure/subscribers/plugin-source.subscriber.ts

```
@@ -73,6 +73,68 @@ export class PluginSourceSubscriber extends BaseEntityEventSubscriber<PluginSour
+	}
+
+	/**
+	 * Generates a full name for a plugin source based on its type and properties
+	 * @param pluginSource The plugin source to generate the full name for
+	 * @returns A descriptive full name for the plugin source
+	 */
+	private generateFullName(source: IPluginSource): string {
+		const { type, operatingSystem, architecture, version } = source;
+
+		// Base parts that are common to all types
+		const baseParts = [`${version ? 'v' + version.number : undefined}`, operatingSystem, architecture];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)